shapeshift/web
# chore: enable onramper flag

## Description



turns flag on for onramper

tested by ops https://discord.com/channels/554694662431178782/1012750280049295360/1022989311173480601

## Notice




## Pull Request Type


## Issue (if applicable)



n/a - related to https://github.com/shapeshift/web/pull/2583

## Risk



minimal - already verified by ops

## Testing



* ensure onramper appears
* ensure all other ramps still appear

### Engineering



### Operations



## Screenshots (if applicable)

<img width="511" alt="image" src="https://user-images.githubusercontent.com/88504456/192312792-1e216e32-a296-4b9a-b553-e677faf955ce.png">
